pkg/cmd/codespace/ports.go

```diff
@@ -87,9 +86,18 @@ func (a *App) ListPorts(ctx context.Context, selector *CodespaceSelector, export
 
 	for _, p := range ports {
 		// filter out internal ports from list
-		if strings.HasPrefix(p.SessionName, vscodeServerPortName) || strings.HasPrefix(p.SessionName, codespacesInternalPortName) {
+		var internal bool
+		for _, tag := range p.Tags {
+			if strings.EqualFold(tag, portforwarder.InternalPortTag) {
+				internal = true
+				break
+			}
+		}
```

Comment:
Just a suggestion, I wonder if this logic is better located in `portforwarder` instead and just exposed as `portforwarder.IsInteralPort(p)`

Reply:
That definitely makes sense, moved!